server/services/courtAppearanceService.ts

```diff
@@ -805,6 +805,8 @@ export default class CourtAppearanceService {
         }
 
         courtAppearance.overallConvictionDate = overallConvictionDate.toDate()
+      } else {
```

Comment:
if overallConvictionDateAppliedAll != 'true'  then reset the overallConvictionDate (this needs to be set correctly for the offence date validation tests in case they set it and then go back ti the screen and unset it)